charts/retool/templates/deployment_backend.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   labels:
     {{- include "retool.labels" . | nindent 4 }}
     {{- include "retool.selectorLabels" . | nindent 4 }}
+{{- if .Values.deploymentLabels }}
```

Comment:
```suggestion
{{- if .Values.deployment.labels }}
```

Reply:
hi! can we do `.labels` instead of `Labels` to keep it consistent with the `.annotations` value below? i can commit & merge if it's ok

Reply:
yes feel free to commit and merge!